src/choropleth.ts

```diff
@@ -267,10 +327,38 @@ function drawLegend(color) {
     .attr("fill", (d: string) => d);
 
   d3.select("div#map-legend-date").text(
-    dateRangeString(latestCountyData["latestDate"])
+    dateRangeString(dateList[selectedDateIndex])
   );
 }
 
+function setDateControlState() {
+  if (selectedDateIndex == 0) {
+    console.log("setDateControlState: A");
```

Comment:
Remove 3 instances of debug console.log.

Reply:
Done